<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserEditWidget</class>
 <widget class="QWidget" name="UserEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edycja</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/biorhythm_16.png</normaloff>:/images/biorhythm_16.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="MyLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Kliknij, aby wyszukać nowy awatar...</string>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/images/user.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButtonDelete">
         <property name="toolTip">
          <string>Usuń awatar (przywróć domyślny)</string>
         </property>
         <property name="text">
          <string>&amp;Usuń</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/image--minus.png</normaloff>:/images/image--minus.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="MyLabel" name="labelAvatars">
         <property name="font">
          <font>
           <underline>true</underline>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Otwórz nowe okno z przykładowymi awatarami</string>
         </property>
         <property name="text">
          <string>Przykładowe awatary...</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="indent">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonImport">
         <property name="toolTip">
          <string>Kliknij, aby wyszukać nowy awatar...</string>
         </property>
         <property name="text">
          <string>&amp;Importuj...</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/image--plus.png</normaloff>:/images/image--plus.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">color: rgb(102, 102, 102);</string>
     </property>
     <property name="text">
      <string>Zalecana rozdzielczość: 48x48 pikseli</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&amp;Nazwa:</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Wpisz nazwę użytkownika widoczną w programie</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>16</number>
       </property>
       <property name="placeholderText">
        <string>Wpisz nazwę użytkownika</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Data urod&amp;zenia:</string>
       </property>
       <property name="buddy">
        <cstring>dateEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit">
       <property name="toolTip">
        <string>Wybierz datę urodzenia</string>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="correctionMode">
        <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>1983</year>
         <month>2</month>
         <day>27</day>
        </datetime>
       </property>
       <property name="displayFormat">
        <string>dd.MM.yyyy</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="selectedDate">
      <date>
       <year>1983</year>
       <month>2</month>
       <day>27</day>
      </date>
     </property>
     <property name="verticalHeaderFormat">
      <enum>QCalendarWidget::NoVerticalHeader</enum>
     </property>
     <property name="navigationBarVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Wybierz użytkownika domyślnie wybieranego podczas uruchamiania programu</string>
       </property>
       <property name="text">
        <string>D&amp;omyślnie wybierany</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelStar">
       <property name="minimumSize">
        <size>
         <width>17</width>
         <height>17</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>17</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/images/default.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="toolTip">
        <string>Zapisz zmiany i wyjdź</string>
       </property>
       <property name="text">
        <string>&amp;Zapisz</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/disk-black.png</normaloff>:/images/disk-black.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="toolTip">
        <string>Wyjdź bez zapisywania zmian</string>
       </property>
       <property name="text">
        <string>&amp;Anuluj</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/cross.png</normaloff>:/images/cross.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header>mylabel.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>pushButtonImport</tabstop>
  <tabstop>pushButtonDelete</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
